<launch>
    <param name="/use_sim_time" value="false"/>
    <node name="relative_loc" pkg="relative_loc" type="relative_loc_node" output="screen" args="$(find relative_loc)/config/hyperparams.yaml" />
</launch>